src/components/UpcomingTalkBox/UpcomingTalkBox.tsx

```diff
@@ -23,22 +28,14 @@ const UpcomingTalkBox: React.FC = () => {
           whiteSpace: "nowrap",
         }}
       >
-        <UpcomingTalkBoxItem />
-        <UpcomingTalkBoxItem />
-        <UpcomingTalkBoxItem />
-        <UpcomingTalkBoxItem />
-        <UpcomingTalkBoxItem />
-        <UpcomingTalkBoxItem />
-        <UpcomingTalkBoxItem />
-        <UpcomingTalkBoxItem />
-        <UpcomingTalkBoxItem />
-        <UpcomingTalkBoxItem />
-        <UpcomingTalkBoxItem />
-        <UpcomingTalkBoxItem />
-        <UpcomingTalkBoxItem />
+        {sessions.length === 0 ? (
+          <UpcomingTalkBoxItem key={"notalk"} title={"No upcoming talks"} />
+        ) : (
+          <UpcomingTalkBoxItem key={sessions[0].id} title={sessions[0].title} />
+        )}
       </div>
     </div>
-  );
-};
+  )
+}
 
-export default UpcomingTalkBox;
+export default UpcomingTalkBox
```

Comment:
An other similar case the semicolon is used. I think it should be used, but be the usage consistent